tracer/build/_build/Build.Steps.cs

```diff
@@ -1189,6 +1242,61 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
             }
         });
 
+    Target RunDebuggerIntegrationTests => _ => _
+        .Unlisted()
+        .After(BuildTracerHome)
+        .After(BuildDebuggerIntegrationTests)
+        .Requires(() => Framework)
+        .Triggers(PrintSnapshotsDiff)
+        .Executes(() =>
+        {
+            EnsureExistingDirectory(TestLogsDirectory);
+            EnsureResultsDirectory(DebuggerIntegrationTests);
+
+            try
+            {
+                DotNetTest(config => config
+                    .SetDotnetPath(TargetPlatform)
+                    .SetConfiguration(BuildConfiguration)
+                    .SetTargetPlatform(TargetPlatform)
+                    .SetFramework(Framework)
+                    .EnableCrashDumps()
+                    .EnableNoRestore()
+                    .EnableNoBuild()
+                    .SetFilter(GetTestFilter())
+                    .SetProcessEnvironmentVariable("MonitoringHomeDirectory", MonitoringHomeDirectory)
+                    .SetLogsDirectory(TestLogsDirectory)
+                    .When(!string.IsNullOrEmpty(Filter), c => c.SetFilter(Filter))
+                    .When(CodeCoverage, ConfigureCodeCoverage)
+                    .EnableTrxLogOutput(GetResultsDirectory(DebuggerIntegrationTests))
+                    .WithDatadogLogger()
+                    .SetProjectFile(DebuggerIntegrationTests));
+
+                string GetTestFilter()
+                {
+                    string filter;
+                    if (IsWin)
+                    {
+                        filter = "RunOnWindows=True";
+                    }
+                    else if (IsArm64)
+                    {
+                        filter = "&(Category!=ArmUnsupported)";
+                    }
+                    else
+                    {
+                        filter = "(Category!=LinuxUnsupported)";
+                    }
+
+                    return Filter is null ? filter : $"{Filter}&{filter}";
+                }
```

Comment:
There's a bug in the arm64 filter, and personally I would use a switch expression 😉 

```suggestion
                string GetTestFilter() => 
                {
                    var filter = (IsWin, IsArm64) switch
                    {
                        (true, _) => "(RunOnWindows=True)",
                        (_, true) => "(Category!=ArmUnsupported)",
                        _ => "(Category!=LinuxUnsupported)",
                    };

                    return Filter is null ? filter : $"{Filter}&{filter}";
                }
```